tests/gameResource.spec.js

```diff
@@ -67,13 +67,13 @@ describe("resource : gameResource", function() {
   });
 
   beforeEach(function() {
-    spyOn(yf, "api").andCallThrough();
+    spyOn(yf, "api").and.callThrough();
```

Comment:
Seems that jasmine [changed their API](https://stackoverflow.com/questions/32599346/how-to-solve-typeerror-spyon-andreturn-is-not-a-function) in version 2. Version 3 is specified in the package.json, so I think we need this syntax.